packages/cog/src/cli/basemaps/README.md

```diff
@@ -0,0 +1,68 @@
+# Basemaps Configuration utility
+
+This tool is designed to configure the basemaps rendering engine prority.
+
+### TileSets
+
+To render a basemap, a collection of imagery needs to be configured, basemaps uses a 'TileSet' for this configuration.
+
+A tileset is a list of imgagery along with the zoom levels to render at and the priority for rendering. The higher the number, the later the imagery will be layered onto the canvas. So bigger number means it will be ontop when viewed.
+
+The entire history of the tileset is stored inside the metadata database and version tags are used to configure what is rendered to the user
+
+Common tags:
+
+-   Head - The most recent tile set configuration
+-   Production - The tile set configuration which is rendered by default to the user
```

Comment:
These appear to be lower case in the code. Suggest writing them here as `head` and `production`